Vostok.Metrics.System/Helpers/SimplifiedIntendedTreeParser.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Vostok.Metrics.System.Helpers
+{
+    #region IntendedTree usage example
+
+    // Example input: 
+    /*
+        setup:
+          runner: activebackup
+        ports:
+          eno1
+            link watches:
+              link summary: down
+              instance[link_watch_0]:
+                name: ethtool
+                link: down
+                down count: 0
+          eno2
+            link watches:
+              link summary: down
+              instance[link_watch_0]:
+                name: ethtool
+                link: down
+                down count: 0
+          enp4s0f0
+            link watches:
+              link summary: up
+              instance[link_watch_0]:
+                name: ethtool
+                link: up
+                down count: 0
+          enp4s0f1
+            link watches:
+              link summary: up
+              instance[link_watch_0]:
+                name: ethtool
+                link: up
+                down count: 0
+        runner:
+          active port: enp4s0f0
+    */
+    // Example parameters: "ports", 2
+    // Example output: 'eno1', 'eno2', 'enp4s0f0', 'enp4s0f1'
+
+    #endregion
+    
+    [Obsolete]
```

Comment:
Current code uses Newtonsoft.JSON to parse driver output.
I don't really like the fact that we have to parse JSON and include Newtonsoft into project, because we don't need whole JSON. We only need a couple of fields. 

Driver outputs JSON which can be parsed using this IndededTreeParser. It is very cheap and simple.

I'm not sure JSON parsing is a reason to worry, so I left there a snippet of simplified parser.

Reply:
I wouldn't bother developing a custom parser since it could become obsolete one day due to slight formatting changes in the response. Sure, Newtonsoft.JSON is a bit too heavy for the task, but it's hidden from users and therefore does not put any obligations on us.